compiler/damlc/tests/src/DA/Test/DamlcIntegration.hs

```diff
@@ -237,7 +252,26 @@ data DiagnosticField
   | DSeverity !DiagnosticSeverity
   | DSource !String
   | DMessage !String
-  deriving (Eq, Show)
+
+renderRange :: Range -> String
```

Comment:
Can we just use `prettyRange` and avoid inconsistencies with how we render it at the command line? Then you also don’t have to worry about whether you merge this before or after the ghcide bump.

Reply:
We could, but we would need to export `prettyRange` from `ghcide`, which it is currently not. Would you prefer that?

Reply:
Ah didn’t realize it wasn’t exposed, I’ll leave it up to you. Seems like a useful cleanup but I also don’t care all that much about it.

Reply:
Given how simple the implementation is and that I expect it to _never_ change again and we have a test checking they don't diverge, I'll leave `prettyRange` internal to `ghcide` and keep a copy here.